src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-inline: auto;
+  }
+
+  &__title {
+    color: $color-black;
+    margin: 0;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    padding: 0;
+    margin: 0;
```

Comment:
I'd recommend list those tags where you need reset default browser styles
Example:
```
body,
h2, 
p {
  margin: 0;
}
```